### PR TITLE
ceph: Update default ceph image to latest v16.2.5

### PR DESCRIPTION
**Pull request by travisn**
**Description of your changes:**
The latest pacific release is 16.2.5 so we set this as the default for Rook to deploy.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: Documentation/ceph-upgrade.md [hunk 1]
```diff
@@ -416,7 +416,7 @@ The majority of the upgrade will be handled by the Rook operator. Begin the upgr
 Ceph image field in the cluster CRD (`spec.cephVersion.image`).
 
 ```sh
-NEW_CEPH_IMAGE='ceph/ceph:v16.2.4-20210514'
+NEW_CEPH_IMAGE='ceph/ceph:v16.2.5-20210514'
```
Comment (BlaineEXE) [inline Documentation/ceph-upgrade.md:419]:
```suggestion
NEW_CEPH_IMAGE='ceph/ceph:v16.2.5-20210708'
```

File: cluster/examples/kubernetes/ceph/cluster.yaml [hunk 2]
```diff
@@ -19,9 +19,9 @@ spec:
     # v13 is mimic, v14 is nautilus, and v15 is octopus.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
     # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
-    # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v16.2.4-20210514
+    # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v16.2.5-20210514
```
Comment (BlaineEXE) [inline cluster/examples/kubernetes/ceph/cluster.yaml:22]:
```suggestion
    # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v16.2.5-20210708
```

File: Documentation/ceph-upgrade.md (hunk 1 above)
Comment (travisn) [reply Documentation/ceph-upgrade.md:419]: good catch, thanks

File: cluster/examples/kubernetes/ceph/cluster.yaml (hunk 2 above)
Comment (BlaineEXE) [reply cluster/examples/kubernetes/ceph/cluster.yaml:22]: What about this one?

**Review by BlaineEXE — CHANGES_REQUESTED** [review]

File: cluster/examples/kubernetes/ceph/cluster.yaml (hunk 2 above)
Comment (travisn) [reply cluster/examples/kubernetes/ceph/cluster.yaml:22]: ugh, ok should have everything in the next push...

**Review by BlaineEXE — APPROVED** [review]

**Pull request merged**